### PR TITLE
render something/anything while we are loading the user session

### PR DESCRIPTION
**Pull request by jchris**
Calling render before we load the remote session is good, because that can take up to 5 seconds. This avoids the white screen of death in that time.

### CODE REVIEW

**Comment by jchris** [comment]
not ready to merge as it has an issue where it doesn't pop the login form when you need one, lemme fix

File: src/authentication/SecretForm.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ class SecretForm extends Component {
 
 const stateToProps = state => ({
   currentUser: state.currentUser,
-  show: !state.currentUser
+  show: !state.lifecycle.restoring && !state.currentUser
```
Comment (erickpintor) [inline src/authentication/SecretForm.js]: Shouldn't we show the pop up but with a "loading session" message?

**Comment by jchris** [comment]
I think that would involve more flashing than we want, since it'll often be
subsecond. If we have a network activity indicator, that might close the
deal.

File: src/app/App.js [hunk 2]
```diff
@@ -61,16 +62,21 @@ const onChangeDatabase = (dispatch, getState) => (nextState, replace, callback)
     dispatch(updateClients(rootClient, splat))
     return callback()
   }
-
+  callback()
```
Comment (marrony) [inline src/app/App.js]: I believe calling the callback here, the next calls would be unnecessary

Comment (marrony) [inline src/app/App.js]: Also, if we want to render the page regardless of the user is restored we can omit the callback from the function and the page will be rendered synchronously

**Comment by jchris** [comment]
I updated it to leave the callback out and it seems to work great.

**Pull request merged**